src/ui/components/inputs/Slider.tsx

```diff
@@ -137,6 +133,21 @@ export function Slider(props: Props) {
     sliderContainerRef.current = ref
   }, [])
 
+  const handleValueChange = useCallback(
+    (nextValues: number[]) => {
+      props.onValuesChange?.(nextValues as ValuesType)
+      setValues(nextValues as ValuesType)
```

Comment:
on devrait pouvoir se passer des `as`

peut etre comme ça

```suggestion
    (nextValues: ValuesType) => {
      props.onValuesChange?.(nextValues)
      setValues(nextValues)
```

sinon en passant par Yup

Reply:
il y a plusieurs occurrences de ce problème

Reply:
https://github.com/pass-culture/pass-culture-app-native/pull/5413#discussion_r1362352341